src/imex.rs

```diff
@@ -439,19 +438,11 @@ pub fn normalize_setup_code(s: &str) -> String {
     out
 }
 
-async fn imex_inner(
-    context: &Context,
-    what: ImexMode,
-    param: Option<impl AsRef<Path>>,
-) -> Result<()> {
-    ensure!(param.is_some(), "No Import/export dir/file given.");
-
-    info!(context, "Import/export process started.");
-    context.emit_event(EventType::ImexProgress(10));
-
+async fn imex_inner(context: &Context, what: ImexMode, path: impl AsRef<Path>) -> Result<()> {
+    info!(context, "Import/export dir: {}", path.as_ref().display());
```

Comment:
```suggestion
    info!(context, "Import/export path: {}", path.as_ref().display());
```

`path` may also be a file in case of `ImexMode::ImportBackup`

Reply:
i use "path" as a reference to directory or file.  Are you suggesting some other naming?

Reply:
as the suggested change: just using "path" also in the comment - "Import/export path: {}" instead of "Import/export dir: {}" :)